app/scripts/BedLikeTrack.js

```diff
@@ -1076,11 +1071,20 @@ class BedLikeTrack extends HorizontalTiled1DPixiTrack {
     this.animate();
   }
 
-  getMouseOverHtml(trackX, trackY) {
+  getMouseOverHtml(trackX, trackY, isShiftDown) {
```

Comment:
What's happening with the shift key?

Reply:
If `showTooltip` is `false`, but the user manually holds down the Shift key, then `isShiftDown` is `true` and this will show the tooltip. This allows view configurations where the tooltip is out of the way of the UI, unless the user holds down the Shift key. I added a comment to this function to explain this, as well.